libcudacxx/examples/concurrent_hash_table.cu

```diff
@@ -13,27 +13,13 @@
 #include <thrust/host_vector.h>
 #include <thrust/iterator/counting_iterator.h>
 #include <thrust/pair.h>
+#include <thrust/system/cuda/vector.h>
 
 #include <cassert>
 #include <cstdio>
 #include <iostream>
 #include <random>
 
-// TODO: This should be upstreamed and then removed.
-namespace thrust
-{
-
-using universal_raw_memory_resource =
-  thrust::system::cuda::detail::cuda_memory_resource<thrust::system::cuda::detail::cudaMallocManaged, cudaFree, void*>;
-
-template <typename T>
-using universal_allocator = thrust::mr::stateless_resource_allocator<T, universal_raw_memory_resource>;
-
-template <typename T>
-using universal_vector = thrust::device_vector<T, universal_allocator<T>>;
-
-} // namespace thrust
-
```

Comment:
This is a drive-by fix, since this functionality has been upstreamed into Thrust IIUC.